src/api/messages/getTemporaryFileUrl.js

```diff
@@ -0,0 +1,24 @@
+/* @flow strict-local */
+import type { Auth, ApiResponseSuccess } from '../transportTypes';
+import { apiGet } from '../apiFetch';
+
+type ApiResponseTempFileUrl = {|
+  ...ApiResponseSuccess,
+  url: string,
+|};
+
+/**
+ * This endpoint, introduced in zulip/zulip@9e72fbe3f, returns a partial URL
```

Comment:
> introduced in zulip/zulip@9e72fbe3f

Eventually, we'll move to including a value of `zulip_feature_level` ([doc](https://zulipchat.com/api/server-settings)) here, and we'll even be able to use that for feature detection in using this feature, as Greg notes at https://github.com/zulip/zulip-mobile/issues/3303#issuecomment-615517986, following #4049, but I think this is fine for now; a commit is certainly as specific as it's possible to be.